WorkloadManagementSystem/DB/ElasticJobDB.py

```diff
@@ -58,7 +61,7 @@ def getJobParameters(self, jobID, paramList=None):
     else:
       query = {"query": {"match": {"JobID": jobID}}, "_source": ["Name", "Value"]}
 
-    gLogger.debug("Getting results for ", jobID)
+    gLogger.debug("Getting results for %d" % jobID)
```

Comment:
This was not a mistake. The logger methods take two arguments and print them both. But the first argument is meant to be "Stable", eventually searchable as it is. The second argument is variable